cico_run_pydoc.sh

```diff
@@ -5,7 +5,7 @@ set -ex
 prep() {
     yum -y update
     yum -y install epel-release
-    yum -y install python34 python34-virtualenv which
+    yum -y install python36
```

Comment:
Don't we need to install `which`? I recall we sometimes have to install it explicitly, so just to be sure... :-)

Reply:
I am pretty sure we don't, it comes with EPEL7 by default (see output bellow).

```terminal
[root@b5af6be000db opt]# rpm -qil which
Name        : which
Version     : 2.20
Release     : 7.el7
Architecture: x86_64
Install Date: Mon Mar 30 16:57:38 2015
Group       : Applications/System
Size        : 76962
License     : GPLv3
Signature   : RSA/SHA256, Fri Jul  4 05:43:07 2014, Key ID 24c6a8a7f4a80eb5
Source RPM  : which-2.20-7.el7.src.rpm
Build Date  : Tue Jun 10 02:25:22 2014
Build Host  : worker1.bsys.centos.org
Relocations : (not relocatable)
Packager    : CentOS BuildSystem <http://bugs.centos.org>
Vendor      : CentOS
URL         : http://www.xs4all.nl/~carlo17/which/
Summary     : Displays where a particular program in your path is located
Description :
The which command shows the full pathname of a specified program, if
the specified program is in your PATH.
/etc/profile.d/which2.csh
/etc/profile.d/which2.sh
/usr/bin/which
/usr/share/doc/which-2.20
/usr/share/doc/which-2.20/AUTHORS
/usr/share/doc/which-2.20/COPYING
/usr/share/doc/which-2.20/EXAMPLES
/usr/share/doc/which-2.20/NEWS
/usr/share/doc/which-2.20/README
/usr/share/info/which.info.gz
/usr/share/man/man1/which.1.gz
[root@b5af6be000db opt]# repoquery -i which

Name        : which
Version     : 2.20
Release     : 7.el7
Architecture: x86_64
Size        : 76962
Packager    : CentOS BuildSystem <http://bugs.centos.org>
Group       : Applications/System
URL         : http://www.xs4all.nl/~carlo17/which/
Repository  : base
Summary     : Displays where a particular program in your path is located
Source      : which-2.20-7.el7.src.rpm
Description :
The which command shows the full pathname of a specified program, if
the specified program is in your PATH.

```

Reply:
cool it must have changed. So please ignore my previous comment :)

Reply:
No problem, thanks for the thorough review, it's priceless!